src/conv/heuristics/ai_heuristics.cpp

```diff
@@ -145,6 +145,47 @@ class Model
     virtual std::vector<float> ToFeatures(const ProblemDescription& problem) const = 0;
 };
 
+class Gfx90aModel : public Model
+{
+public:
+    Gfx90aModel() : Model("gfx90a") {}
+    bool IsProblemSupported(const ProblemDescription& problem,
+                            const ConvolutionContext& ctx) const override
```

Comment:
```suggestion
    bool IsProblemSupported(const ProblemDescription&,
                            const ConvolutionContext&) const override
```
I guess HIP Tidy complains about unused parameters.

Reply:
They are going to be used, I just haven't added the code yet. This PR is in draft.